pkgs/applications/window-managers/e16/default.nix

```diff
@@ -51,6 +51,9 @@ stdenv.mkDerivation rec {
 
   postPatch = ''
     substituteInPlace scripts/e_gen_menu --replace "/usr/local:" "/run/current-system/sw:/usr/local:"
+    substituteInPlace scripts/e_gen_menu --replace "/usr/bin/python3" "/run/current-system/sw/bin/env python3"
```

Comment:
```suggestion
```
Just add `python3` to `buildInputs`

Reply:
Thank you for the review & suggestion - I've made this change and added `python3` to `buildInputs`.